ymer/src/test/java/com/avanza/ymer/ConverterTest.java

```diff
@@ -115,21 +330,65 @@ public void setFile(String file) {
 			this.file = file;
 		}
 
-		public LocalDateTime getTime() {
-			return time;
+		public LocalDateTime getLocalDateTimeNano() {
+			return localDateTimeNano;
+		}
+
+		public void setLocalDateTimeNano(LocalDateTime localDateTimeNano) {
+			this.localDateTimeNano = localDateTimeNano;
+		}
+
+		public Instant getInstant() {
+			return instant;
 		}
 
-		public void setTime(LocalDateTime time) {
-			this.time = time;
+		public void setInstant(Instant instant) {
+			this.instant = instant;
 		}
 
+		public LocalDate getLocalDate() {
+			return localDate;
+		}
+
+		public void setLocalDate(LocalDate localDate) {
+			this.localDate = localDate;
+		}
 		@Override
 		public String toString() {
 			return "SpaceFilePublicationEvent{" +
 					"id='" + id + '\'' +
 					", file='" + file + '\'' +
-					", time='" + time + '\'' +
+					", localDateTimeNano='" + localDateTimeNano + '\'' +
+					", instant='" + instant + '\'' +
+					", localDate='" + localDate + '\'' +
+					", currency='" + currency + '\'' +
+					", yearMonth='" + yearMonth + '\'' +
+					", localDateTimeMillis='" + localDateTimeMillis + '\'' +
 					'}';
 		}
+
+		public Currency getCurrency() {
+			return currency;
+		}
+
+		public void setCurrency(Currency currency) {
+			this.currency = currency;
+		}
+
+		public YearMonth getYearMonth() {
+			return yearMonth;
+		}
+
+		public void setYearMonth(YearMonth yearMonth) {
+			this.yearMonth = yearMonth;
+		}
+
+		public LocalDateTime getLocalDateTimeMillis() {
+			return localDateTimeMillis;
+		}
```

Comment:
Great that you extended the tests so thoroughly! ⭐ 👍

Reply:
Thanks! It was necessary to really understand which converters that could be deprecated and not.

Reply:
I agree, nice with good test coverage ⭐